warehouse/query-core/src/test/java/datawave/query/jexl/visitors/SetMembershipVisitorTest.java

```diff
@@ -18,17 +25,152 @@
 
 public class SetMembershipVisitorTest {
     
-    private final ShardQueryConfiguration config = new ShardQueryConfiguration();
+    private static ShardQueryConfiguration config;
     private final Set<String> fields = new HashSet<>();
     private JexlNode query;
     
+    private static Set<String> indexOnly;
+    private static MockMetadataHelper helper;
+    private static MockDateIndexHelper helper2;
+    
+    @BeforeClass
+    public static void initializeMetadata() throws Exception {
+        indexOnly = Sets.newHashSet("INDEX_ONLY", "ONLY_INDEXED", "BODY");
+        helper = new MockMetadataHelper();
+        helper.setIndexedFields(indexOnly);
+        helper.setIndexOnlyFields(indexOnly);
+        helper.addTermFrequencyFields(Collections.singleton("BODY"));
+        
+        helper2 = new MockDateIndexHelper();
+        config = new ShardQueryConfiguration();
+        config.setDatatypeFilter(Collections.singleton("test"));
+        config.setLazySetMechanismEnabled(true);
+        
+    }
+    
     @After
     public void tearDown() throws Exception {
         config.setLazySetMechanismEnabled(false);
         query = null;
         fields.clear();
     }
     
+    @Test
+    public void simpleTestMembers() throws Exception {
+        Assert.assertEquals(Sets.newHashSet("INDEX_ONLY"),
+                        SetMembershipVisitor.getMembers(indexOnly, config, JexlASTHelper.parseJexlQuery("INDEX_ONLY == 'foobar'")));
+    }
```

Comment:
What do you think about having the tests delegate to a test method, something like this?
```java
public void simpleTestMembers(){
    test("INDEX_ONLY == 'foobar'", Sets.newHashSet("INDEX_ONLY"));
}

private void test(String query, Set<String> expectedMembers){
    // query parsing and asserts
}
```

Reply:
Sure, I can make that adjustment in the next commit